### DISCUSSION TITLE
Updating mechanical design

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#93] [Show and tell]
After feedback from the field, I decided to put some time on revamping the mechanical design:

![Screenshot from 2022-01-18 17-19-34](https://user-images.githubusercontent.com/1565933/150046159-b1149ac2-7b66-49ef-a53b-f5284811030a.png)

**Comment by peterantypas** [comment]
The two main changes are:

- The antenna cap and the antenna former will thread into each other with M18-2.5 threads (coarse enough to be reliably 3D printed)
- An optional adapter will allow MAIANA to mount on standard 1"-14 marine antenna mounts. We'll find out soon enough if that thread forms in ASA material. If not, there's always CNC :wink:

**Comment by peterantypas** [comment]
The cap and antenna former will have these slots. I'm hoping that they will allow the cap to press-fit tightly into the casing by deforming slightly when pushed in.

![Screenshot from 2022-01-18 17-53-05](https://user-images.githubusercontent.com/1565933/150048926-11ddbf1c-6fe6-424e-9443-f7774b4d266a.png)

If this works out, there will be no need for a heat shrink process to hold these two together. Also, a bit of Locktite on the thread should help.

**Comment by kagouraki** [comment]
I dont think the threads will work. With that, you make things more complex and you dont gain something special. 
The slot seems a bit big. I would recommend 4 of 0.5mm slot all around if you want to proceed this idea. 
The think that need definetly improoval is some geometry to hold the pcb in the tube. Now, it is holded only be the sma connector which i dont think is a good solution.

**Reply by peterantypas** [reply]
The board is supported on all 4 sides (top, sides and bottom), as it is going to be flush against the walls of the enclosure and the mounting adapter will jam against the bottom and Cat5 jack. Also, the SMA connector is now intentionally somewhat loose and flexible to avoid undue stress.

**Comment by peterantypas** [comment]
The 1"-14 antenna mount made of ASA worked, but process tolerances made it a little too tight and it needed some tapping. I should be able to dial those down over the next 1-2 iterations.

![IMG_1642](https://user-images.githubusercontent.com/1565933/151672462-ce5e2d47-3c21-44b0-b809-71e65871bc2b.JPG)

**Reply by peterantypas** [reply]
The main challenge with some of these "ratchet" mounts is that they may or may not pass an RJ45 connector (the one above does, barely). Straight mounts like this one have no issues:

![Screenshot from 2022-01-29 10-14-30](https://user-images.githubusercontent.com/1565933/151672572-114c3656-4fdb-4682-849e-8301c046ec4f.png)

**Comment by peterantypas** [comment]
I had success with 3D printing TPU (thermoplastic urethane) to form a cable seal. The initial test was promising, but more harsh testing will be necessary. The idea is that the mounting adapter also comes with a grommet that wraps around the Cat5 cable, then gets compressed like a gland during assembly:

![IMG_1644](https://user-images.githubusercontent.com/1565933/152575022-7ad484ab-421e-4950-835c-a7cc095246d2.jpg)

I'm hoping that with this and some kind of insulation layer, no condensation will ever form inside the tube even in the most harsh environment. MAIANA already generates a certain amount of heat at the RF circuits. If I can trap this heat well enough, that should do the trick.

**Comment by Rudder2** [comment]
I like this.  One of my fears is water would get in from the bottom.

**Comment by peterantypas** [comment]
Success! Iteration #2 fits perfectly. Still 1-2 iterations to go.

![IMG_1654](https://user-images.githubusercontent.com/1565933/153527681-81413d56-28ab-4200-8a3d-9404913aac7d.JPG)

**Comment by peterantypas** [comment]
For the next iteration, I'm going to try creating a lip in the adapter for the seal to snap into. I hope this is it.

![Screenshot from 2022-02-14 15-36-29](https://user-images.githubusercontent.com/1565933/153965166-66551ef5-9012-4fd6-a70a-9dd83b24c7ff.png)

**Comment by peterantypas** [comment]
Here is the 3d iteration of the antenna mount. Mechanically everything fits as expected. It's not waterproof, of course, but I'm not going for IP6x rating, just IP5x. 

Now that I've seen it in action, I think that a higher ratio heat shrink should be able to seal the bottom better. Will try this next week.

![IMG_1684](https://user-images.githubusercontent.com/1565933/155893067-a8570774-07f8-444e-b4e1-7afd9048b642.JPG)

**Comment by peterantypas** [comment]
I had the new prototypes made of ASA instead of PLA. The threads formed well, and I think that's it. This material is much stronger and offers great UV resistance, so it's appropriate for outdoor use. I will start working on the assembly manual.
